apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/BrevRoute.kt

```diff
@@ -182,11 +182,18 @@ fun Route.brevRoute(
                 logger.info("Oppretter nytt brev på sak=$sakId)")
                 val brevParametre = call.receive<BrevParametre>()
 
+                val spraak =
+                    when (brevParametre) {
+                        is BrevParametre.Aktivitetsplikt -> null
+                        is BrevParametre.TomtBrev -> Spraak.NB
```

Comment:
Blir litt dumt for de med grunnlag og et annet språk, kanskje burde man egentlig se på krr her?